ERCS/erc-7802.md

```diff
@@ -0,0 +1,259 @@
+---
+eip: 7802
+title: Crosschain Token Interface
+description: Token interface that allows authorized contracts to mint and burn token representations during cross-chain transfers
+author: skeletor (@skeletor-spaceman), parti (@0xParticle), joxes (@Joxess), ng (@0xng), agus duha (@agusduha), disco (@0xDiscotech), gotzen <gotzen@defi.sucks>, 0age <0age@uniswap.org>, Mark Tyneway <mark@oplabs.co>, Zain Bacchus <zain@oplabs.co>, Matt Solomon <msolomon@oplabs.co>, Maurelian <maurelian@protonmail.ch>, Blaine Malone (@blmalone)
+discussions-to: https://ethereum-magicians.org/t/erc-7802-crosschain-token-interface/21508
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-30
+requires: 165, 5679
+---
+
+## Abstract
+
+This standard introduces a minimal interface, `IERC7802`, for tokens to enable standardized cross-chain communication. The interface consists of two functions, `crosschainMint` and `crosschainBurn`, which allow authorized bridge contracts to mint and burn token representations during cross-chain transfers. These functions serve as the entry points for bridge logic, enabling consistent handling of token supply across chains.
+
+The interface also defines two standardized events, `CrosschainMint` and `CrosschainBurn`, which emit metadata, including the target address, token amount, and caller. These events facilitate deterministic indexing and monitoring of cross-chain activities by off-chain agents, such as indexers, analytics tools, and auditors.
```

Comment:
Please choose either cross-chain or crosschain and use it consistently throughout the document.